vassal-app/src/main/java/VASSAL/counters/UsePrototype.java

```diff
@@ -251,6 +257,53 @@ public Editor(UsePrototype up) {
       nameConfig = new StringConfigurer(up.type.substring(ID.length()));
       controls.add("Editor.UsePrototype.prototype_name", nameConfig);
 
+      final JButton selectButton = new JButton(Resources.SELECT);
```

Comment:
You need to use the key to get the value here. I get a button that says `General.select`.

Reply:
"Oops"